user/hare/template.py

```diff
@@ -0,0 +1,42 @@
+pkgname = "hare"
+pkgver = "0.24.0"
+pkgrel = 0
+archs = ["aarch64", "riscv64", "x86_64"]
+build_style = "makefile"
+make_env = {"VERSION": pkgver, "LOCALVER": "chimera"}
+match self.profile().arch:
+    case "x86_64":
+        _qbe_arch = "amd64_sysv"
+    case "aarch64":
+        _qbe_arch = "arm64"
+    case "riscv64":
+        _qbe_arch = "rv64"
+    case _:
+        _qbe_arch = ""
+make_build_args = [
+    f"ARCH={self.profile().arch}",
+    f"QBEFLAGS=-t{_qbe_arch}",
+    f"{self.profile().arch.upper()}_CC=cc",
+    f"{self.profile().arch.upper()}_LD=ld",
+]
+# TODO: make it use clang assembler (at build time and also at runtime)
+hostmakedepends = [f"binutils-{self.profile().arch}", "harec", "qbe", "scdoc"]
+depends = ["binutils", "clang", "harec", "qbe", "tzdata"]
+checkdepends = ["tzdata"]
+pkgdesc = "Hare programming language"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "MPL-2.0 AND GPL-3.0-only"
+url = "https://harelang.org"
+source = f"https://git.sr.ht/~sircmpwn/hare/archive/{pkgver}.tar.gz"
+sha256 = "7061dad3c79cca51a1662a71b1c6f8ec001f52ef3053dd3c2dbb95ae9beff7bc"
+tools = {"AS": f"{self.profile().triplet}-as"}
```

Comment:
maybe we can also have `hare-{arch}` subpackages for cross-building like we have for binutils?

Reply:
sure, see the alpine install of hare that adds config files for other arches

Reply:
ah you added that already

hm, maybe then..

Reply:
maybe i'm overthinking this, but installing e.g. `hare-riscv64` will also pull in `clang` and `binutils` for the host arch, we might want to make it possible to install the deps for just a given arch without also installing the ones for the host arch

Reply:
ok i ended up having to add `hare-{arch}` anyway, to make the list of subpackages consistent across architectures